lib/wificom/mqtt.py

```diff
@@ -40,16 +41,29 @@ def connect_to_mqtt(mqtt_client):
 	_mqtt_client.on_unsubscribe = unsubscribe
 
 	# Connect to MQTT Broker
-	print("Connecting to MQTT Broker...")
-	# _io.connect()
-	_mqtt_client.connect()
+	attempt = 0
+	while attempt < 3:
+		try:
+			print(f"Connecting to MQTT Broker (attempt {attempt+1})...")
+			_mqtt_client.connect()
+
+			return _mqtt_client
```

Comment:
Would returning True/False make sense in this function? We don't need to return the same object that was passed in.